app/controllers/application_controller.rb

```diff
@@ -30,19 +29,19 @@ def options
     
 protected
   
-  def rescue_action(env)
+  def rescue_action(_env)
     respond_to do |wants|
       wants.json { render :json => "Error", :status => 500 }
     end
   end
 
-  def cors_set_access_control_headers
+  def cors_set_access_control_headers # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
10 counts of Metrics/MethodLength too. Seems like it previously detected 22 as the threshold. Maybe call it 20 and see if we can reduce the peppering of disables?: